src/components/lib/FlowGrid/edge.js

```diff
@@ -95,16 +95,17 @@ export default class Edge extends Component{
   }
 
   render() {
-    const {output, input, hasErrors, pathStatus} = this.props
+    const {output, input, hasErrors, pathStatus, className} = this.props
     if (!this._isValidNode(input) || !this._isValidNode(output)) { return (false) }
     let inputPoints = (new Rectangle(input)).showPosition(output)
     let outputPoints = (new Rectangle(output)).showPosition(input)
 
     let points = `M${inputPoints.x},${inputPoints.y} L${outputPoints.x} ,${outputPoints.y}`
 
+    console.log(className, this.props)
     return (
         <path
-            className={`basic-arrow ${pathStatus}${hasErrors ? ' hasErrors' : ''}`}
+            className={`basic-arrow ${pathStatus}${' ' + className}${hasErrors ? ' hasErrors' : ''}`}
```

Comment:
Check out the `getClassName` helper in utils to clean this section up. You can do something like:
```
className={getClassName('basic-arrow', className, hasErrors ? 'hasErrors' : null)}
```